src/app/component/AddToCart/AddToCart.container.js

```diff
@@ -123,27 +119,34 @@ export class AddToCartContainer extends PureComponent {
         }).then(() => this._afterAdded());
     }
 
+    removeProductFromWishlist() {
```

Comment:
You are returning mixed type, either null, or a function. Is this intended? Please make this function void, if this was not intended.